### ISSUE TITLE
32 Bit restriction on a 64 bit os.

### ISSUE THREAD

**Issue by 0rdinal** [TheRemote/RaspberryPiMinecraft#10]
**Referring to [this](https://github.com/TheRemote/RaspberryPiMinecraft/blob/efbdb15db6049507d05605e23e2b83b34efb47ab/SetupMinecraft.sh#L29-L72) piece of code**

SetupMinecraft.sh detects the os as 32bit when It is actually 64bit

**Console log:**
```
Getting total system memory...
Total memory: 3808 - Available Memory: 3624
Total memory: 3808 - Available Memory: 3624
Please enter the amount of memory you want to dedicate to the server.  A minimum of 700MB is recommended.
You must leave enough left over memory for the operating system to run background processes.
If all memory is exhausted the Minecraft server will either crash or force background processes into the paging file (very slow).
Enter amount of memory in megabytes to dedicate to the Minecraft server (recommended: 3624): 3624
You are running a 32 bit operating system which has a limit of 2700MB.  Please enter 2700 to use it all.
You can lift this restriction by upgrading to a 64 bit operating system.
```

I jumped through the code and you assign $CPUArch to the response of `uname-m` which for this os returns:
```aarch64```

**Comment by benjamw** [comment]
If you remove lines 65-68 you can get around this. The check isn't even looking at the arch, just assuming 32-bit.

**Comment by Randomblock1** [comment]
Fixed in #12. Tested on 64-bit kernel Raspbian on RPi 4.

**Comment by TheRemote** [comment]
Fixed from the pull request, thanks!
